src/WWT.Providers/Providers/TwoMassToastProvider.cs

```diff
@@ -27,55 +31,19 @@ public override void Run(IWwtContext context)
                 context.Response.End();
                 return;
             }
-
-            if (level < 9)
-            {
-                try
-                {
-                    context.Response.ContentType = "image/png";
-                    Stream s = PlateTilePyramid.GetFileStream(wwtTilesDir + "\\2MassToast0to7.plate", level, tileX, tileY);
-                    int length = (int)s.Length;
-                    byte[] data = new byte[length];
-                    s.Read(data, 0, length);
-                    context.Response.OutputStream.Write(data, 0, length);
-                    context.Response.Flush();
-                    context.Response.End();
-                    return;
-                }
-                catch
-                {
-                    context.Response.Clear();
-                    context.Response.ContentType = "text/plain";
-                    context.Response.Write("No image");
-                    context.Response.End();
-                    return;
-                }
-
-            }
             else
```

Comment:
This branch was never hit. If `level > 7`, the first branch was hit. The second branch took everything else as it has the condition `level < 9`. There are no remaining conditions for the final block to exist.